services/nvidia-gpu-operator/22.9.1/defaults/cm.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: ${releaseNamespace}
 data:
   values.yaml: |-
+    priorityClassName: "dkp-high-priority"
```

Comment:
here's the helm chart values: https://github.com/NVIDIA/gpu-operator/blob/master/deployments/gpu-operator/values.yaml I see that they are already defaulting to using `system-node-critical`, which I think is probably the right thing to do here (I don't think we should override them) so let's actually not make any changes here! I'll make a note in the doc about this

```suggestion
```